### PR TITLE
[1938] Viewing dependencies of a new article page with an uploaded image displays dependencies of image instead of page

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/1938

### CODE REVIEW

File: static-assets/components/cstudio-view-controllers/dependencies.js [hunk 1]
```diff
@@ -99,13 +99,15 @@
...
     CStudioAuthoring.Service.loadItems(callback, data);
 
-    $(document).on("keyup", function (e) {
+    $(document).on('keyup', function (e) {
       if (e.keyCode === 27) {	// esc
         me.end();
-        $(document).off("keyup");
+        $(document).off('keyup');
       }
     });
```
Comment (rart) [inline static-assets/components/cstudio-view-controllers/dependencies.js]: Off the handler, not the event.

**Review by rart — CHANGES_REQUESTED** [review]
Please test generously.

**Review by rart — CHANGES_REQUESTED** [review]
If the dialog is not closed via escape key, the listener still remains active. 
- Listen for the end event of the dialog to also destroy the event listener
- Make sure you remove the listener **before** you call on me.end on the escape listener
- Test

**Comment by jvega190** [comment]
@rart I made a little refactor, I added a wrong commit message, but the commit is for the last comment you added

**Review by rart — APPROVED** [review]

**Pull request merged**